application-services/custom/cloud-event/pkg/transforms/conversion_test.go

```diff
@@ -186,7 +188,15 @@ func TestTransformToCloudEventMultipleEvents(t *testing.T) {
 	for i, cloudEvent := range cloudEvents {
 		resultBytes, err := json.Marshal(cloudEvent)
 		require.NoError(t, err)
-		assert.Equal(t, want[i], string(resultBytes))
+		wantMap := map[string]string{}
+		err = json.Unmarshal([]byte(want[i]),&wantMap)
+		require.NoError(t, err)
+		actualMap := map[string]string{}
+		err = json.Unmarshal(resultBytes,&actualMap)
+		require.NoError(t, err)
+		if !reflect.DeepEqual(wantMap,actualMap) {
+			t.Fatalf("Return value doesn't match expected.\nExpected: %v\nActual: %v\n", want[i], string(resultBytes))
+		}
```

Comment:
assert.Equal does a deep equal check

```suggestion
		assert.Equal(t,wantMap,actualMap) 
```